src/Http/Routing/WebRouting.php

```diff
@@ -97,6 +99,11 @@ public function wire(): void
             $this->router->post("/newsletter/unsubscribe", "destroy")->name("newsletter.destroy");
         });
 
+        $this->router->controller(InvitationsController::class)->group(function (): void{
```

Comment:
The most of controllers is in singular name. Maybe rename it to `InvitationController`?